firebase-perf/src/main/java/com/google/firebase/perf/util/FirstDrawDoneListener.java

```diff
@@ -0,0 +1,115 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+//
+// You may obtain a copy of the License at
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package com.google.firebase.perf.util;
```

Comment:
I don't know where else to drop this in. I don't want to drop it in `/metrics` which is where `AppStartTrace` and other Trace metrics live in, because that would allow `AppStartTrace` access to some package-private methods that should not be allowed to use apart from testing. I only want `AppStartTrace` (the consumer of `FirstDrawDoneListener`) to use public static factory to register and create `FirstDrawDoneListener`.